websites/G/GenoAnime/presence.ts

```diff
@@ -0,0 +1,58 @@
+const presence = new Presence({
+  clientId: "810203651317432351"
+}),
+browsingStamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+  const presenceData: PresenceData = {
+    largeImageKey: "geno",
+    details: "Genoanime !!!"
+  },
+title = document.title; //title of the page
+
+  if (document.location.pathname == "/" ) {
+    presenceData.startTimestamp = browsingStamp;
+    presenceData.details = "Exploring Genoanime";
+	presenceData.buttons = [{label:"Visit Genoanime",url: document.location.href}];
+  } else if (document.location.pathname.includes("/browse")) {
+    presenceData.startTimestamp = browsingStamp;
+    presenceData.details = "Exploring Genoanime library";
+	presenceData.buttons = [{label:"See the Library",url: document.location.href}];
```

Comment:
Remove spaces

Reply:
removed spaces and changed to View Library